src/main/scala/org/ergoplatform/network/ModeFeature.scala

```diff
@@ -0,0 +1,73 @@
+package org.ergoplatform.network
+
+import com.google.common.primitives.Ints
+import org.ergoplatform.nodeView.state.StateType
+import org.ergoplatform.settings.NodeConfigurationSettings
+import scorex.core.network.PeerFeature
+import scorex.core.network.PeerFeature.Id
+import scorex.core.serialization.Serializer
+import scorex.core.utils
+
+import scala.util.Try
+
+case class ModeFeature(stateType: StateType,
+                       verifyingTransactions: Boolean,
+                       popowBootstrapping: Boolean,
+                       popowSuffix: Int,
+                       blocksToKeep: Int) extends PeerFeature {
+
+  override val featureId: Id = 16: Byte
+
+  def booleanToByte(bool: Boolean): Byte = if (bool) 1: Byte else 0: Byte
+
+  def byteToBoolean(byte: Byte): Boolean = if (byte > 0) true else false
+
+  override def serializer: Serializer[PeerFeature] = new Serializer[PeerFeature] {
```

Comment:
We usually implement separate object like `object ModeFeatureSerializer extends Serializer[PeerFeature]`, I think it's better to follow this convension.
Also methods `booleanToByte` and `byteToBoolean` should not be in PeerFeature class

Reply:
fixed